src/bp-core/admin/bp-core-admin-tools.php

```diff
@@ -871,6 +871,232 @@ function bp_core_admin_debug_information( $debug_info = array() ) {
 		)
 	);
 
+	$debug_info['buddypress-constants'] = array(
+		'label'       => __( 'BuddyPress Constants', 'buddypress' ),
+		'description' => __( 'These constants can alter where and how parts of BuddyPress are loaded or works.', 'buddypress' ),
+		'fields'      => array(
+			'BP_VERSION' => array(
+				'label' => 'BP_VERSION',
+				'value' => BP_VERSION,
+			),
+			'BP_DB_VERSION' => array(
+				'label' => 'BP_DB_VERSION',
+				'value' => BP_DB_VERSION,
+			),
+			'BP_REQUIRED_PHP_VERSION' => array(
+				'label' => 'BP_REQUIRED_PHP_VERSION',
+				'value' => BP_REQUIRED_PHP_VERSION,
+			),
+			'BP_PLUGIN_DIR' => array(
+				'label' => 'BP_PLUGIN_DIR',
+				'value' => BP_PLUGIN_DIR,
+			),
+			'BP_PLUGIN_URL' => array(
+				'label' => 'BP_PLUGIN_URL',
+				'value' => BP_PLUGIN_URL,
+			),
+			'BP_IGNORE_DEPRECATED' => array(
+				'label' => 'BP_IGNORE_DEPRECATED',
+				'value' => defined( 'BP_IGNORE_DEPRECATED' ) && BP_IGNORE_DEPRECATED ? __( 'Enabled', 'buddypress' ) : __( 'Disabled', 'buddypress' ),
+				'debug' => defined( 'BP_IGNORE_DEPRECATED' ) ? BP_IGNORE_DEPRECATED : 'undefined',
+			),
+			'BP_LOAD_DEPRECATED' => array(
+				'label' => 'BP_LOAD_DEPRECATED',
+				'value' => defined( 'BP_LOAD_DEPRECATED' ) && BP_LOAD_DEPRECATED ? __( 'Enabled', 'buddypress' ) : __( 'Disabled', 'buddypress' ),
+				'debug' => defined( 'BP_LOAD_DEPRECATED' ) ? BP_LOAD_DEPRECATED : 'undefined',
+			),
+			'BP_FORUMS_PARENT_FORUM_ID' => array(
```

Comment:
Sorry I missed this one (`BP_FORUMS_PARENT_FORUM_ID`), it should be moved to the bottom with the "(deprecated") mention you added for other deprecated constants.

Reply:
Moved! 💪🏾